docs/en/connector-v2/sink/Druid.md

```diff
@@ -0,0 +1,101 @@
+# Druid
+
+> Druid sink connector
+
+## Description
+
+Write data to Apache Druid.
+
+
+## Options
+
+| name                    | type     | required | default value |
+| ----------------------- | -------- | -------- | ------------- |
+| coordinator_url         | `String` | yes      | -             |
+| datasource              | `String` | yes      | -             |
+| columns                 | `List<String>` | yes| __time        |
+| timestamp_column        | `String` | no       | timestamp     |
+| timestamp_format        | `String` | no       | auto          |
+| timestamp_missing_value | `String` | no       | -             |
+
+### coordinator_url [`String`]
```

Comment:
Please use the same format as other documents  https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/connector-v2/sink/Clickhouse.md